wunderio/next4drupal-project
# NEX-50: Add modal component

<img width="1052" alt="CleanShot 2023-05-18 at 15 03 20@2x" src="https://github.com/wunderio/next4drupal-project/assets/22575651/1c0a3a5c-009c-403b-88d9-75d390b00b35">

*Changes proposed in this PR:*
- Add Modal component to wunder-component-library (based off the design in Figma)
- Update Storybook to latest

*How to test:*
1. Checkout this branch
2. Check the feature environment of this branch still works.  Note that you won't find the modal anywhere there - it was added to Storybook but is not yet used in the site itself.
3. `lando npm i && lando npm run storybook`, check the Modal works well there

*Link to feature environment:*
https://feature-nex-50-modal-next.next4drupal-project.dev.wdr.io
